### ISSUE TITLE
[Feature Request] Download to Directory Automatically

### ISSUE THREAD

**Issue by Worldgate** [BlafKing/sd-civitai-browser-plus#16]
Would it be possible to have the 'destination folder' have a dynamic naming?  Basically I like to keep the folder cleaned up so if i move files out that I'm not using their named in a way that makes sense to me.

An example would be: [https://civitai.com/models/134643/blacklight-makeup-sdxl-lora]

blacklight-makeup-sdxl-lora-134643

I just use the directory name from the website and then append the model number because sometimes there are multiple releases of the same lora and i like to keep them separate.

**Comment by BlafKing** [comment]
That should definitely be possible!

I think an easy way of implementing this is by adding it automatically as options in the sub folder list.
I'll implement it a bit different than you're suggestion, I'll probably make it so that 2 options get added to the subfolder list by default being:
```
{model}
{model}\{version}
```

so with your example it would insert these 2 options into the sub folder droplist:
```
Blacklight Makeup — SDXL LoRA
Blacklight Makeup — SDXL LoRA\v2
```

I think it's best to do it this way since not all models use a filename which correlates well to the actual model and also since the API doesn't return the link of the models

**Comment by BlafKing** [comment]
The functionality mentioned above is now implemented in v1.8.1 !
I'll close this thread now, thanks for the request! :)
